docs/challenges/memory/pte-reorder-paragraphs.md

```diff
@@ -0,0 +1,157 @@
+# PTE Reorder Paragraphs
```

Comment:
can you rename this file challenge_b.md ? 
It's just the way we name things right now, we'll change it later.

and update the mkdocs.yml ?
```
    - List of Challenges:
      - Memory:
        - Introduction: challenges/memory/introduction.md
        - Memory Challenge A: challenges/memory/challenge_a.md
        - Memory Challenge B: challenges/memory/challenge_b.md
        - Memory Challenge C: challenges/memory/challenge_c.md
        
```